lib/nrf_modem_lib/nrf_modem_os.c

```diff
@@ -12,14 +12,21 @@
 #include <nrf.h>
 #include <nrf_errno.h>
 #include <errno.h>
-#include <pm_config.h>
+
 #include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(nrf_modem, CONFIG_NRF_MODEM_LIB_LOG_LEVEL);
+
+#if CONFIG_SOC_SERIES_NRF91X
+#include <pm_config.h>
+#define SHMEM_TX_HEAP_ADDR (PM_NRF_MODEM_LIB_TX_ADDRESS)
+#elif CONFIG_SOC_SERIES_NRF92X
+#define SHMEM_TX_HEAP_ADDR (DT_REG_ADDR(DT_NODELABEL(cpuapp_cpucell_ipc_shm_heap)))
+#endif
+#define SHMEM_TX_HEAP_SIZE (CONFIG_NRF_MODEM_LIB_SHMEM_TX_SIZE)
```

Comment:
Should the size be given by DT for nRF92?

Reply:
Yes, I have updated this.